RobotCode/src/main/java/frc/robot/commands/ToggleIntakeCommand.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class ToggleIntakeCommand extends Command
+{
+
+	private final IntakeSubsystem Intake;
+
+	public ToggleIntakeCommand(IntakeSubsystem intake)
+	{
+		this.Intake = intake;
+	}
+
+	@Override
+	public void initialize()
+	{
+		Intake.raiseOrLowerIntake();
+	}
+
+	@Override
+	public void end(boolean stop)
+	{
+		Intake.stopIntake();
+	}
+
+	@Override
+	public boolean isFinished()
+	{
+		return Intake.isStopped();
```

Comment:
This check won't work. You stop the intake in end, but end doesn't run until the intake is stopped.